README.md

```diff
@@ -12,7 +12,7 @@ $ npm install debug
 
 ## Usage
 
- With `debug` you simply invoke the exported function to generate your debug function, passing it a name which will determine if a noop function is returned, or a decorated `console.error`, so all of the `console` [format string goodies](https://developer.chrome.com/devtools/docs/console-api#consolelogobject-object) you're used to work fine. A unique color is selected per-function for visibility.
+`debug` exposes a function; simply pass this function the name of your module, and it will return a decorated version of `console.error` for you to pass debug statements to. This will allow you to specify which modules you wish to see debug statements from when working an issue.
```

Comment:
Can we reword `This will allow you to specify which modules you wish to see debug statements from when working an issue.` to `This will allow you to toggle on and off log output for different parts of your project`